### PR TITLE
cradle-sql: add page

### PR DESCRIPTION
**Pull request by pxgamer**
See #2560

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/cradle-sql.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Rebuild the database schema:
+
+`cradle sql build`
+
+- Rebuild the database schema for a specific package:
+
+`cradle sql build {{package}}`
+
+- Truncate the entire database:
```
Comment (sbrl) [inline pages/common/cradle-sql.md]: Doe  this just do a cleanup (like the SQLite `VACUUM;` command), or actually remove records?

Comment (pxgamer) [reply pages/common/cradle-sql.md]: It removes all records, i.e. a ``TRUNCATE TABLE `table_name`;``

Comment (sbrl) [reply pages/common/cradle-sql.md]: I see. Perhaps `Empty the entire database` would be clearer?

Comment (pxgamer) [reply pages/common/cradle-sql.md]: Sounds ok to me 👍

**Pull request merged**